src/core/sys/darwin/netinet/in_.d

```diff
@@ -0,0 +1,605 @@
+    enum IPPROTO_DDP    = 37;
+    enum IPPROTO_CMTP   = 38;
+    enum IPPROTO_TPXX   = 39;
+    enum IPPROTO_IL     = 40;
+}
+enum IPPROTO_IPV6 = 41;
+static if (_DARWIN_C_SOURCE)
+{
+    enum IPPROTO_SDRP      = 42;
+    enum IPPROTO_ROUTING   = 43;
+    enum IPPROTO_FRAGMENT  = 44;
+    enum IPPROTO_IDRP      = 45;
+    enum IPPROTO_RSVP      = 46;
+    enum IPPROTO_GRE       = 47;
+    enum IPPROTO_MHRP      = 48;
+    enum IPPROTO_BHA       = 49;
+    enum IPPROTO_ESP       = 50;
+    enum IPPROTO_AH        = 51;
+    enum IPPROTO_INLSP     = 52;
+    enum IPPROTO_SWIPE     = 53;
+    enum IPPROTO_NHRP      = 54;
+    enum IPPROTO_ICMPV6    = 58;
+    enum IPPROTO_NONE      = 59;
+    enum IPPROTO_DSTOPTS   = 60;
+    enum IPPROTO_AHIP      = 61;
+    enum IPPROTO_CFTP      = 62;
+    enum IPPROTO_HELLO     = 63;
+    enum IPPROTO_SATEXPAK  = 64;
+    enum IPPROTO_KRYPTOLAN = 65;
+    enum IPPROTO_RVD       = 66;
+    enum IPPROTO_IPPC      = 67;
+    enum IPPROTO_ADFS      = 68;
+    enum IPPROTO_SATMON    = 69;
+    enum IPPROTO_VISA      = 70;
+    enum IPPROTO_IPCV      = 71;
+    enum IPPROTO_CPNX      = 72;
+    enum IPPROTO_CPHB      = 73;
+    enum IPPROTO_WSN       = 74;
+    enum IPPROTO_PVP       = 75;
+    enum IPPROTO_BRSATMON  = 76;
+    enum IPPROTO_ND        = 77;
+    enum IPPROTO_WBMON     = 78;
+    enum IPPROTO_WBEXPAK   = 79;
+    enum IPPROTO_EON       = 80;
+    enum IPPROTO_VMTP      = 81;
+    enum IPPROTO_SVMTP     = 82;
+    enum IPPROTO_VINES     = 83;
+    enum IPPROTO_TTP       = 84;
+    enum IPPROTO_IGP       = 85;
+    enum IPPROTO_DGP       = 86;
+    enum IPPROTO_TCF       = 87;
+    enum IPPROTO_IGRP      = 88;
+    enum IPPROTO_OSPFIGP   = 89;
+    enum IPPROTO_SRPC      = 90;
+    enum IPPROTO_LARP      = 91;
+    enum IPPROTO_MTP       = 92;
+    enum IPPROTO_AX25      = 93;
+    enum IPPROTO_IPEIP     = 94;
+    enum IPPROTO_MICP      = 95;
+    enum IPPROTO_SCCSP     = 96;
+    enum IPPROTO_ETHERIP   = 97;
+    enum IPPROTO_ENCAP     = 98;
+    enum IPPROTO_APES      = 99;
+    enum IPPROTO_GMTP      = 100;
+    enum IPPROTO_PIM       = 103;
+    enum IPPROTO_IPCOMP    = 108;
+    enum IPPROTO_PGM       = 113;
+    enum IPPROTO_SCTP      = 132;
+    enum IPPROTO_DIVERT    = 254;
+}
+enum IPPROTO_RAW = 255;
+
+static if (_DARWIN_C_SOURCE)
+{
+    enum IPPROTO_MAX  = 256;
+    enum IPPROTO_DONE = 257;
+}
+
+static if (_DARWIN_C_SOURCE)
+{
+    enum IPPORT_RESERVED = 1024;
+    enum IPPORT_USERRESERVED = 5000;
+
+    enum IPPORT_HIFIRSTAUTO = 49152;
+    enum IPPORT_HILASTAUTO  = 65535;
+
+    enum IPPORT_RESERVEDSTART = 600;
+}
+
+static if (_DARWIN_C_SOURCE)
+{
+    extern(D) bool IN_CLASSA(in_addr_t i) pure @safe { return (i & 0x80000000) == 0; }
+    enum IN_CLASSA_NET    = 0xff000000;
+    enum IN_CLASSA_NSHIFT = 24;
+    enum IN_CLASSA_HOST   = 0x00ffffff;
+    enum IN_CLASSA_MAX    = 128;
+
+    extern(D) bool IN_CLASSB(in_addr_t i) pure @safe { return (i & 0xc0000000) == 0x80000000; }
```

Comment:
All of these macros use `u_int32_t` instead of `in_addr_t `. They all correspond to the same type though.